src/v0/destinations/tiktok_ads/data/TikTokTrack.json

```diff
@@ -73,8 +73,18 @@
     "required": false
   },
   {
-    "destKey": "context.page",
-    "sourceKeys": ["properties.context.page", "context.page"],
+    "destKey": "context.ad.callback",
+    "sourceKeys": "properties.clickId",
+    "required": false
+  },
+  {
```

Comment:
As we are unable to test out whether tiktok allows usage on variables other than url & referrer, let's get it reviewed with product & then roll out the changes